File: WebApplication/src/modules/student/dtos/person-lot.dtos/person-student.dtos/person-student.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { BaseStudentDoctoralDto } from "../base.dtos/base-student-doctoral.dto";
import { PersonStudentDiplomaDto } from "./person-student-diploma.dto";
import { StudentSemesterDto } from "./student-semester.dto";

export class PersonStudentDto extends BaseStudentDoctoralDto {
  facultyNumber: string;
  qualification: string;
  semesters: StudentSemesterDto[];
  school: NomenclatureDto;
  schoolSettlement: NomenclatureDto;
  diploma: PersonStudentDiplomaDto;
}

File: StudentCard.Application/Students/Dtos/PersonLotDtos/PersonStudentDtos/PersonStudentDto.cs
using StudentCard.Application.Students.Dtos.PersonLotDtos.BaseDtos;
using StudentCard.Infrastructure.Helpers.Dtos;
using System.Collections.Generic;

namespace StudentCard.Application.Students.Dtos.PersonLotDtos.PersonStudentDtos
{
    public class PersonStudentDto : BaseStudentDoctoralDto
    {
        public string FacultyNumber { get; set; }

        public string Qualification { get; set; }

        public PersonStudentDiplomaDto Diploma { get; set; }

        public List<StudentSemesterDto> Semesters { get; set; } = new List<StudentSemesterDto>();

        public NomenclatureDto School { get; set; }
        public NomenclatureDto SchoolSettlement { get; set; }
    }
}
